### PR TITLE
Adding Desktop to System.Globalization.Extensions

### PR DESCRIPTION
**Pull request by venkat-raman251**
@weshaggard : Please have a look.

### CODE REVIEW

File: src/System.Globalization.Extensions/src/System.Globalization.Extensions.csproj [hunk 1]
```diff
@@ -19,17 +20,14 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'OSX_Release|AnyCPU' " />
```
Comment (weshaggard) [inline src/System.Globalization.Extensions/src/System.Globalization.Extensions.csproj]: Can you remove the spaces at the start and end of these conditions for consistency

**Comment by weshaggard** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test

**Comment by weshaggard** [comment]
Other than the one clean-up it looks good.

**Comment by venkat-raman251** [comment]
@weshaggard : Cleaned up trailing/leading spaces.

**Pull request merged**